lib/cluster.js

```diff
@@ -27,7 +28,19 @@ function Worker(options) {
   if (options === null || typeof options !== 'object')
     options = {};
 
-  this.suicide = undefined;
+  this.voluntaryExit = undefined;
+
+  Object.defineProperty(this, 'suicide', {
+    get: function() {
+      // TODO: Print deprecation message.
+      return this.voluntaryExit;
+    },
+    set: function(val) {
+      // TODO: Print deprecation message.
```

Comment:
Should this message include information about when (date or version) to expect deprecation to become removal?